accounts/models.py

```diff
@@ -657,16 +659,29 @@ def send_as_email(self):
         self.save()
 
 
-def create_subject_for_study_notification(study, children):
-    latter_half = f' invited to take part in "{study.name}" on Lookit!'
+def create_string_listing_children(children):
     child_names = [child.given_name for child in children]
+    num_children = len(child_names)
```

Comment:
apologies for not using `:=` here - precommit was balking at it even as already used?

Reply:
How were you trying to use it?

Reply:
oh I see what you're saying - yeah, that's strange. If our precommit environments are identical that shouldn't happen, unless something actually changed in the code itself that would invalidate the syntax.

Reply:
on that note - make sure your precommit and virtualenv are both using 3.8.x (I think our pipenv env will force 3.8.3 on you)